makefiles/suit.v4.inc.mk

```diff
@@ -10,6 +10,15 @@ SUIT_MANIFEST_LATEST ?= $(BINDIR_APP)-riot.suitv4.latest.bin
 SUIT_MANIFEST_SIGNED ?= $(BINDIR_APP)-riot.suitv4_signed.$(APP_VER).bin
 SUIT_MANIFEST_SIGNED_LATEST ?= $(BINDIR_APP)-riot.suitv4_signed.latest.bin
 
+SUIT_URLPATH ?= "$(SUIT_COAP_BASEPATH)/$(shell echo $$(basename $(SUIT_MANIFEST_SIGNED_LATEST)))"
+SUIT_URLPATH_LEN ?= $(shell expr length \"$(SUIT_URLPATH)\")
+SOCK_URLPATH_MAXLEN ?= $(shell echo "$$((($(SUIT_URLPATH_LEN) / 32 + 1 ) * 32))")
+ifeq ($(shell expr $(SOCK_URLPATH_MAXLEN) \>= 128), 1)
+   export CFLAGS += -DSOCK_URLPATH_MAXLEN=128
```

Comment:
IMO we can just use this...

Reply:
Would you be ok with an error message if its too long then?

Reply:
The function states that when failing. So scratch that.